.github/styles/Vocab/Docs/accept.txt

```diff
@@ -87,20 +91,19 @@ Nagios
 namespace
 namespaced
 (nginx|NGINX)
-NVIDIA
-Nvidia
-nvidia
+(NVIDIA|Nvidia|nvidia)
 onboarding
 overfit(|s|ted|ting)
 Papermill
 Percona
 PostgreSQL
 preprocessor(|s)
-Prometheus
+(Prometheus|Prometheus)
```

Comment:
```suggestion
(Prometheus|prometheus)
```

Reply:
the base branch was changed Matt so this is not a valid change anymore.